Formula/fuego-firestore.rb

```diff
@@ -0,0 +1,18 @@
+class FuegoFirestore < Formula
```

Comment:
Why not just call it `fuego`? That's the name everywhere else

Reply:
There is already a `fuego` formula:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/fuego.rb